gtsfm/utils/metrics.py

```diff
@@ -128,7 +128,7 @@ def compute_translation_angle_metric(
     for (i1, i2) in i2Ui1_dict:
         i2Ui1 = i2Ui1_dict[(i1, i2)]
         angles.append(comp_utils.compute_translation_to_direction_angle(i2Ui1, wTi_list[i2], wTi_list[i1]))
-    return GtsfmMetric("translation_angle_deg", angles)
+    return GtsfmMetric("translation_angle_deg", np.array(angles, dtype=np.float))
```

Comment:
why do we need the cast here? GtsfmMetric should allow accepting List[float]

Reply:
For the nan values. Its actually List[Optional[float]].